src/ParseConfig.js

```diff
@@ -95,10 +95,10 @@ class ParseConfig {
    * @return {Promise} A promise that is resolved with a newly-created
    *     configuration object or with the current with the update.
    */
-  static save(attrs: { [key: string]: any }) {
+  static save(attrs: { [key: string]: any }, masterKeyOnly: { [key: string]: any }) {
     const controller = CoreManager.getConfigController();
     //To avoid a mismatch with the local and the cloud config we get a new version
-    return controller.save(attrs).then(() => {
+    return controller.save(attrs, masterKeyOnly).then(() => {
       return controller.get();
```

Comment:
You should pass useMasterKey into get(). That ways after you save you will get the current version. You can write a test case for this as well.

Reply:
Added test case